### PR TITLE
Fix #4904 unread-PMs count blank when too big

### LINKED ISSUE #4904: Unread-PMs count is blank if too big
The count we show on the "people" icon that leads to the PMs tab can look like this:
image
I.e., there's a little bubble there… but instead of having a number inside, it's just blank, and so it looks broken.

I get that when logged in as a test account that has 139 unread PMs. I suspect what's happening involves the fact that the number is too long to really fit in that space.

Instead, when this count is big, we should probably just say something like "99+".

### PR DESCRIPTION
**Pull request by ShristiJain**
Fixes: #4904

### CODE REVIEW

File: src/chat/UnreadNotice.js [hunk 1]
```diff
@@ -30,6 +30,11 @@ const styles = createStyleSheet({
     color: 'white',
     paddingRight: 4,
   },
+  unreadNumberExceeds: {
+      fontSize: 12,
```
Comment (chrisbobbe) [inline src/chat/UnreadNotice.js:34]: It looks like this new `styles.unreadNumberExceeds` is just meant to decrease the font size, from 14 to 12.

Can we find a solution that doesn't require an inconsistent font size? If the problem is that you're running into a maximum width of that `BRAND_COLOR`ed bubble, can we instead remove that constraint, and let the width grow just enough to show "99+" without shrinking the text?

File: src/chat/UnreadNotice.js [hunk 2]
```diff
@@ -30,6 +30,11 @@ const styles = createStyleSheet({
     color: 'white',
     paddingRight: 4,
   },
+  unreadNumberExceeds: {
```
Comment (chrisbobbe) [inline src/chat/UnreadNotice.js:33]:
```
Issue #4904 Solved
```

Please follow our [commit message style](https://github.com/zulip/zulip-mobile/blob/master/docs/style.md#git-commits-commit-messages).

**Review by chrisbobbe — CHANGES_REQUESTED** [review]
Thanks, @ShristiJain! Comments below.

File: src/chat/UnreadNotice.js (hunk 2 above)
Comment (ShristiJain) [reply src/chat/UnreadNotice.js:33]: Thankyou for pointing this out. I have made the edit.

Comment (chrisbobbe) [reply src/chat/UnreadNotice.js:33]: Ah, I see you've changed the title, but I was talking about your first comment in the PR, [this one](https://github.com/zulip/zulip-mobile/pull/4979#issue-722716437). I think GitHub also calls this the "description" of the PR.

**Comment by chrisbobbe** [comment]
Ah, and I see GitHub is pointing out some failed checks: https://github.com/zulip/zulip-mobile/pull/4979/checks?check_run_id=3465031609

It says `FAILED: prettier`. You should be able to fix that with `tools/test prettier --fix`.

**Comment by ShristiJain** [comment]
hey @chrisbobbe I am new to this open source contribution. It would be great if you could help me with where exactly can I fix this issue of FAILED: prettier.

**Comment by chrisbobbe** [comment]
> hey @chrisbobbe I am new to this open source contribution. It would be great if you could help me with where exactly can I fix this issue of FAILED: prettier.

`FAILED: prettier` means there's some code that isn't formatted correctly. You can also see that output by running the command `tools/test prettier`, or just `tools/test`.

`tools/test prettier --fix` will try to fix the problem. 🙂 What output do you get from that command? Does `git status` show that some formatting corrections have been made?

I think `tools/fmt` would also work.

**Comment by ShristiJain** [comment]
> > hey @chrisbobbe I am new to this open source contribution. It would be great if you could help me with where exactly can I fix this issue of FAILED: prettier.
> 
> `FAILED: prettier` means there's some code that isn't formatted correctly. You can also see that output by running the command `tools/test prettier`, or just `tools/test`.
> 
> `tools/test prettier --fix` will try to fix the problem. 🙂 What output do you get from that command? Does `git status` show that some formatting corrections have been made?
> 
> I think [`tools/fmt`](https://github.com/zulip/zulip-mobile/blob/master/docs/howto/editor.md#other-editors) would also work.

'tools/test prettier' shows 'passed' in my laptop but fails on Github CI. Git status doesn't show any new changes either. What should I do now?

**Comment by chrisbobbe** [comment]
> 'tools/test prettier' shows 'passed' in my laptop but fails on Github CI. Git status doesn't show any new changes either.

It sounds like there may be some commits that you haven't pushed yet. Could you paste the output of `git show`? Currently, I see GitHub showing d7e721175 as the tip of the remote branch. If that's different from what you see on your local branch, you'll probably want to [force-push](https://zulip.readthedocs.io/en/latest/git/using.html#force-push-changes-to-github-after-you-ve-altered-your-history) your latest changes.

**Comment by ShristiJain** [comment]
> > 'tools/test prettier' shows 'passed' in my laptop but fails on Github CI. Git status doesn't show any new changes either.
> 
> It sounds like there may be some commits that you haven't pushed yet. Could you paste the output of `git show`? Currently, I see GitHub showing [d7e7211](https://github.com/zulip/zulip-mobile/commit/d7e721175394853268334d5d4984caf83eb503cb) as the tip of the remote branch. If that's different from what you see on your local branch, you'll probably want to [force-push](https://zulip.readthedocs.io/en/latest/git/using.html#force-push-changes-to-github-after-you-ve-altered-your-history) your latest changes.

image
It shows the same number!

**Comment by gnprice** [comment]
Thanks @ShristiJain for this contribution! I agree with @chrisbobbe's comment above:
https://github.com/zulip/zulip-mobile/pull/4979#discussion_r698715684

Would you please also post screenshots of what this icon looks like when the count is large, both before and after your change?

**Comment by gnprice** [comment]
After #5147, the code this applies to has been replaced. If the issue still exists, the solution will look quite different from this.

**Pull request closed without merging**